drivers/audio/Kconfig

```diff
@@ -79,7 +79,7 @@ config VS1053_DEVICE_COUNT
 
 config VS1053_WORKER_STACKSIZE
 	int "VS1053 worker thread stack size"
-	default 768
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.